xm-commons-mail/src/test/java/com/icthh/xm/commons/mail/provider/MailProviderServiceUnitTest.java

```diff
@@ -0,0 +1,137 @@
+package com.icthh.xm.commons.mail.provider;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+
+import java.util.Properties;
+
+public class MailProviderServiceUnitTest {
+
+    public static final String TENANT = "TEST";
+
+    private static final String MAIL_CONFIG = "mail-provider:\n"
```

Comment:
Propose to move this in test resource file

Reply:
do you mean just save content to file and add code for reading it from file in test?
i mean this is unit test, so xm-config files are not reading by App itself.

Reply:
In separate file it will more readable and maintainable